griptape/drivers/vector/pgvector_vector_store_driver.py

```diff
@@ -86,15 +87,18 @@ def upsert_vector(
             obj = session.merge(obj)
             session.commit()
 
-            return str(obj.id)
+            return str(getattr(obj, "id"))
```

Comment:
What was the error that prompted this change?

Reply:
`Cannot access member "id" for type "_symbol"`